### ISSUE TITLE
normalize.css -> modern-normalize

### ISSUE THREAD

**Issue by pavelloz** [redwoodjs/sprout#48]
Hello,

Have you thought about switching normalize.css to modern normalize?
Ive seen many modern projects switching, and it seems like the approach is pretty in line with your target group (devs usually dont use ie 11 nowadays :) )

Repo: https://github.com/sindresorhus/modern-normalize

**Comment by cannikin** [comment]
Do we have an explicit include of normalize? I thought that was included by Tailwind itself? Is it possible to override with something else?

**Comment by pavelloz** [comment]
Ouh, i think i made a mistake. [There is a file that contains normalize.css comment](https://github.com/redwoodjs/sprout/blob/main/web/src/scaffold.css), but everything below is not normalize at all - i got fooled :)

I think you are right about Tailwind - it has normalize, and to be precise, modern-normalize that i proposed, so all good. 
https://tailwindcss.com/docs/preflight#overview

Sorry for confusion.

**Comment by cannikin** [comment]
I think it may have started as normalize but then we customized it a bit (prefixed with `.rw-scaffold`). If you have any improvements feel free to PR!

**Comment by pavelloz** [comment]
Im far too fresh in the codebase yet to suggest something, need a little bit more time, until then I will contribute on the subject that im very confident in - performance :)

**Comment by jtoar** [comment]
> Should I create PRs if previous ones are not merged?

TL;DR yes, but you're right in that we need to get better at moving the PRs we have forward. But still, if you want to improve something, by all means, please make a PR!
